Functions/Functions/User/ConnectUserCommentsFunction.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using OidcApiAuthorization.Abstractions;
+using OidcApiAuthorization.Models;
+using System.Linq;
+
+namespace SSW.Rules.Functions
+{
+    public class ConnectUserCommentsFunction
+    {
+        private readonly RulesDbContext _dbContext;
+        private readonly IApiAuthorization _apiAuthorization;
+
+        public ConnectUserCommentsFunction(RulesDbContext dbContext, IApiAuthorization apiAuthorization)
+        {
+            _dbContext = dbContext;
+            _apiAuthorization = apiAuthorization;
+        }
+
+        [FunctionName("ConnectUserCommentsFunction")]
+        public async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+
+            ApiAuthorizationResult authorizationResult = await _apiAuthorization.AuthorizeAsync(req.Headers);
+
+            if (authorizationResult.Failed)
+            {
+                log.LogWarning(authorizationResult.FailureReason);
+                return new UnauthorizedResult();
+            }
+
+            log.LogWarning($"HTTP trigger function {nameof(ConnectUserCommentsFunction)} request is authorized.");
+
+            User data;
+
+            string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+            data = JsonConvert.DeserializeObject<User>(requestBody);
+
+            bool isNull = string.IsNullOrEmpty(data?.CommentsUserId) || string.IsNullOrEmpty(data?.UserId);
+            if (data == null || isNull)
+            {
+                return new BadRequestObjectResult(new
+                {
+                    message = "Request body is empty or incorrect",
+                });
+            }
+            User result;
```

Comment:
What is this for?

Reply:
If the body, or any part of the body is empty, respond with this message using a 400 status code.

Reply:
Sorry, I meant the "User result;", I highlighted the code sections badly when doing the pull request, GitHub doesn't seem to let you change it after the fact.